MetaMask/metamask-extension
# fix: Sentry app state null data to show null as value.

## **Description**
Updating the sentry app state to show a null value when the data is null when the property is not masked.




[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26522?quickstart=1)

## **Related issues**

Fixes:
#25959 

## **Manual testing steps**

 Run `this.stateHooks.getSentryAppState()` in the console and check if the unmasked null values are stored as null.


## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
